pkg/test/utils/kubernetes.go

```diff
@@ -1071,6 +1071,32 @@ func RunCommand(ctx context.Context, namespace string, cmd harness.Command, cwd
 	return nil, err
 }
 
+// toAssertCommands converts a set of TestAssertCommand to Commands so it all the existing functions can be used 
+// that expect Commands data type.
+func toAssertCommands(assertCommands []harness.TestAssertCommand, timeout int) (commands []harness.Command) {
```

Comment:
This does not below in `kubernetes.go` if it goes in `utils` at all...  `utils` predates you (and me) and would prefer better packaging... don't let that block you... but it shouldn't be in `kubernetes`.   This is more kuttl specific behavior

Reply:
I can move that function to the file you tell me. the thing is `kubernetes.go` is already coupled with "harness" namespace and all the utils are using the data model from API. If it would be a 1:1 mapping I would suggest yo move it to `api` package, but since I'm adding some "logic" (background false, timeout..) to make it compatbible with the existing method signatures, I am not clear what would be the "right" place. Maybe in Steps.go. 
 My natural automatic reaction would be make a pakage with all the run utils and it's own data structures, but that is clearly beyond the scope of this PR.
Let me know what is your choice.

Reply:
yeah... I need to dig in.... I was hoping not to add any more coupling there.  It's not too significant... there is a lot of refactoring I would like to do... I don't want to place that burden on you or this PR